tfx/components/example_gen/driver.py

```diff
@@ -92,11 +92,11 @@ def _prepare_output_artifacts(
     example_artifact.set_string_custom_property(
         utils.FINGERPRINT_PROPERTY_NAME,
         exec_properties[utils.FINGERPRINT_PROPERTY_NAME])
-    example_artifact.set_string_custom_property(
+    example_artifact.set_int_custom_property(
```

Comment:
note, this is breaking change, we shall still use string custom property for this?

/cc jyzhao@ any context of changing this to int?

Reply:
@1025KB

Reply:
Hui, we change to int because internal we use int?

or you suggest to keep the string custom property and later we add int property(not custom property) directly?

Reply:
>  you suggest to keep the string custom property and later we add int property(not custom property) directly?

that looks a better approach. I think to unblock the cl, let's keep using string for now, and we can always do it in separate cls. wdyt?

Reply:
sgtm. reverted back to using `set_string_custom_property()`. util functions like `calculate_splits_fingerprint_span_and_version()` still return ints, but `_prepare_output_artifacts()` in `driver.py` coverts int back to string when setting artifact property.